nautobot/circuits/views.py

```diff
@@ -127,8 +136,44 @@ class CircuitUIViewSet(NautobotUIViewSet):
     queryset = Circuit.objects.all()
     serializer_class = serializers.CircuitSerializer
     table_class = tables.CircuitTable
-    # NOTE: This is how `NautobotUIViewSet` would define use_new_ui attr
-    # use_new_ui = ["list", "retrieve"]
+    object_detail_content = ObjectDetailContent(
+        tabs=[
+            ObjectDetailMainTab(
+                layouts=[
+                    LayoutTwoColumn(
+                        include_template_extensions=True,
+                        left_panels=(
+                            ObjectFieldsPanel(
+                                fields="__all__",
+                                exclude_fields=["comments", "circuit_termination_a", "circuit_termination_z"],
+                                field_transforms={"commit_rate": [humanize_speed, placeholder]},
+                            ),
+                            ObjectFieldsPanel(
+                                label="Comments",
+                                fields=["comments"],
+                                field_transforms={"comments": [render_markdown, placeholder]},
+                            ),
+                        ),
+                        right_panels=(
+                            TemplatePanel(
+                                label="Termination - A Side",
+                                # TODO: edit/swap/delete/detail buttons in panel header
+                                content_wrapper=TemplatePanel.ATTR_TABLE_CONTENT_WRAPPER,
+                                template_string="{% include 'circuits/inc/circuit_termination_fragment.html' with termination=circuit_termination_a %}",
+                            ),
```

Comment:
We have already declared it in the template circuit_retrieve.html right? Do we have to have this here? Or vice versa?

Reply:
`circuit_termination.html` isn't used in the detail view any more (it's replaced by this code). I've left it in place in case of the unlikely possibility that some app is relying on it.

Reply:
understood